zebra-rpc/src/methods.rs

```diff
@@ -664,18 +677,39 @@ where
                 })?;
 
             match response {
+                #[cfg(feature = "getblocktemplate-rpcs")]
+                mempool::Response::FullTransactions(mut transactions) => {
+                    transactions.sort_by_cached_key(|tx| {
```

Comment:
Can you please document the sort order here?

I think it's something like:
```suggestion
                    // Sort transactions in descending order by fee/size, using hash in serialized byte order as a tie-breaker
                    transactions.sort_by_cached_key(|tx| {
```